gateway/src/apicast/policy_pusher.lua

```diff
@@ -0,0 +1,59 @@
+local policy_manifests_loader = require('apicast.policy_manifests_loader')
+local http_ng = require('resty.http_ng')
+
+local cjson = require('cjson')
+
+local setmetatable = setmetatable
+local format = string.format
+
+local _M = {}
+
+local mt = { __index = _M }
+
+function _M.new(http_client, manifests_loader)
+  local self = setmetatable({}, mt)
+  self.http_client = http_client or http_ng.new()
+  self.policy_manifests_loader = manifests_loader or policy_manifests_loader
+  return self
+end
+
+local system_endpoint = '/admin/api/registry/policies'
+
+local function system_url(access_token, admin_portal_domain)
+  return format('https://%s:%s%s', access_token, admin_portal_domain, system_endpoint)
+end
+
+local function push_to_system(name, version, manifest, admin_portal_domain, access_token, http_client)
+  local url = system_url(access_token, admin_portal_domain)
+
+  return http_client.post(
+    url, { name = name, version = version, schema = cjson.encode(manifest) }
```

Comment:
You probably want back JSON, so you should send `Accept: application/json` and sending there JSON, so should send `Content-Type: application/json`.

Reply:
Or there should be `http_ng.json.post` that does it.

Reply:
You are right. Fixed :+1: